hvac/api/auth/__init__.py

```diff
@@ -1,7 +1,9 @@
 """Collection of classes for various Vault auth methods."""
 
 from hvac.api.auth.github import Github
+from hvac.api.auth.mfa import Mfa
```

Comment:
this may end up conflicting with https://github.com/hvac/hvac/pull/244/files#diff-6584ecf07e2d3f55775b4244648aa47d once it's merged in

whether it does or not, I get to appear thoughtful and wise